buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -120,6 +120,7 @@ class BuildPlugin implements Plugin<Project> {
                 println "  JDK Version           : ${gradleJavaVersionDetails}"
                 println "  JAVA_HOME             : ${gradleJavaHome}"
             }
+            println " Random Testing Seed    : ${project.testSeed}"
```

Comment:
I wonder if the test seed should actually be created in RandomizedTestingPlugin? Then we can expect it to be here, but it will automatically be pulled in for any gradle project here using randomized testing, regardless if it uses `elasticsearch.build`, eg standalone test.